docs/modules/reference/pages/telemetryd/protocols/bmp/setup.adoc

```diff
@@ -0,0 +1,60 @@
+
+= BMP Setup
+:description: Learn how to set up a BGP Monitoring Protocol (BMP) integration (core, Minions, and Sentinel) in OpenNMS Horizon/Meridian.
+
+This section describes how to set up BMP integration for advanced monitoring of the Border Gateway Protocol, with the core, Minions, and Sentinel.
+
+== Requirements
+
+* BMP-enabled router
+
+== Configure BMP in telemetryd
+
+To configure BMP, enable the BMP-TCP-5000 listener and the various BMP adapters in the `telemetryd-configuration.xml` file, as in the following example for the BMP peer status adapter:
+
+[source, xml]
+----
+     <queue name="BMP">
+        <adapter name="BMP-Peer-Status-Adapter" class-name="org.opennms.netmgt.telemetry.protocols.bmp.adapter.BmpPeerStatusAdapter" enabled="true">
+        </adapter>
+----
+
+In addition to enabling the listener and adapters, you can configure various parameters for the parser and adapters.
+See subsequent chapters in this section for details on these optional parameters.
+
+== Configure BMP Listener on a Minion
+
+To enable and configure a TCP listener for BMP on Minion, connect to the Karaf Console and set the following properties:
+
+NOTE: You must also enable the protocol on {page-component-title} for the messages to be processed.
+
+.From the Minion Karaf console
+[source, karaf]
+----
+config:edit --alias tcp-5000 --factory org.opennms.features.telemetry.listeners
+config:property-set name BMP
+config:property-set class-name org.opennms.netmgt.telemetry.listeners.TcpListener
+config:property-set parameters.port 5000
+config:property-set parsers.0.name BMP
+config:property-set parsers.0.class-name org.opennms.netmgt.telemetry.protocols.bmp.parser.BmpParser
+config:update
+----
+
+If you do not specify the queue name, the fallback is the name of the parser.
+
+== Distributed BMP processing with Sentinel
+
+You can use Sentinel to process BMP statistics ingested by one or more Minions in order to alleviate load on the core.
+All of the BMP-related adapters can run on Sentinel, allowing you to generate events and metrics, and ingest route changes.
+In this mode, events are sent back to the message broker for the core to process and metrics are persisted directly to the time series database.
+
+.From the Sentinel Karaf console
+[source, karaf]
+----
+config:
+config:
+config:
+config:update
+----
+
+//please provide content or correct process.
```

Comment:
I don't have the right setup for this now.   But this doc seems to have the right content https://docs.opennms.com/horizon/32/reference/telemetryd/protocols/bmp/persisting-adapter.html